articles/active-directory/hybrid/how-to-connect-health-agent-install.md

```diff
@@ -39,6 +39,7 @@ The following table lists requirements for using Azure AD Connect Health.
 | Firewall ports on the server are running the agent. |The agent requires the following firewall ports to be open so that it can communicate with the Azure AD Connect Health service endpoints: <br /><li>TCP port 443</li><li>TCP port 5671</li> <br />The latest version of the agent doesn't require port 5671. Upgrade to the latest version so that only port 443 is required. For more information, see [Hybrid identity required ports and protocols](./reference-connect-ports.md).  |
 | If Internet Explorer enhanced security is enabled, allow specified websites.  |If Internet Explorer enhanced security is enabled, then allow the following websites on the server where you install the agent:<br /><li>https:\//login.microsoftonline.com</li><li>https:\//secure.aadcdn.microsoftonline-p.com</li><li>https:\//login.windows.net</li><li>https:\//aadcdn.msftauth.net</li><li>The federation server for your organization that's trusted by Azure AD (for example, https:\//sts.contoso.com)</li> <br />For more information, see [How to configure Internet Explorer](https://support.microsoft.com/help/815141/internet-explorer-enhanced-security-configuration-changes-the-browsing). If you have a proxy in your network, then see the note that appears at the end of this table.|
 | PowerShell version 5.0 or newer is installed. | Windows Server 2016 includes PowerShell version 5.0. 
+| For ADDS and ADFS agents: FIPS (Federal Information Processing Standard) is disabled. | FIPS is only supported on Health agent for Sync since AAD Connect version 2.0.91.0. However is still not supported for Health agent for ADDS and ADFS |
```

Comment:
```suggestion
| For ADDS and ADFS agents: FIPS (Federal Information Processing Standard) is disabled. | FIPS is only supported on Health agent for Sync since Azure AD Connect version 2.0.91.0. However, is still not supported for Health agent for ADDS and ADFS. |
```